libraries/standard/mqtt/cbmc/proofs/MQTT_Init/MQTT_Init_harness.c

```diff
@@ -25,16 +25,17 @@
  */
 
 #include "mqtt.h"
+#include "mqtt_cbmc_state.h"
 
 void harness()
 {
-    MQTTContext_t context;
-    MQTTTransportInterface_t transportInterface;
-    MQTTApplicationCallbacks_t callbacks;
-    MQTTFixedBuffer_t networkBuffer;
+    MQTTContext_t * pContext = mallocCanFail( sizeof( MQTTContext_t ) );
+    MQTTTransportInterface_t * pTransportInterface = mallocCanFail( sizeof( MQTTContext_t ) );
+    MQTTApplicationCallbacks_t * pCallbacks = mallocCanFail( sizeof( MQTTApplicationCallbacks_t ) );
+    MQTTFixedBuffer_t * pNetworkBuffer = mallocCanFail( sizeof( MQTTFixedBuffer_t ) );
```

Comment:
This is different from the style you use above of initializing pointers to NULL at declaration, then assigning the struct pointer below.  I thought the former was the group style.

Reply:
I will change back to the other style for consistency.